x-pack/plugin/rank-rrf/src/internalClusterTest/java/org/elasticsearch/xpack/rank/rrf/RRFRankCoordinatorCanMatchIT.java

```diff
@@ -206,10 +207,10 @@ public void testCanMatchCoordinator() throws Exception {
                 )
                 .setSize(5),
             response -> {
-                assertNull(response.getHits().getTotalHits());
+                assertEquals(new TotalHits(0, TotalHits.Relation.EQUAL_TO), response.getHits().getTotalHits());
```

Comment:
Interesting, why was it null before? We were unskipping one shard, yet getting null total hits?

Reply:
It's because of SearchPhaseController#merge when we skip all the shards we have an empty result and therefore we `return SearchResponseSections.EMPTY_WITH_TOTAL_HITS;`. I wonder if this should instead be `EMPTY_WITHOUT_TOTAL_HITS`.